SPEC/FILES.md

```diff
@@ -161,9 +161,9 @@ pull(
 
 ##### `Go` **WIP**
 
-##### `JavaScript` - ipfs.files.cat(ipfsPath, [callback])
+##### `JavaScript` - ipfs.files.cat(ipfsPath, [begin], [end], [callback])
```

Comment:
Why args like this? Let's add `options` and have that begin and end there.

Reply:
Fair enough, will change.  I've noticed that we have a [`ipfs.files.read`](https://github.com/ipfs/interface-ipfs-core/blob/master/SPEC/FILES.md#read) function that accepts `offset` and `count` - it might be better to use that instead of `begin` and `end` to be consistent.  What do you think?

Also, I can't find an implementation of that in js-ipfs but it looks like it could just call through to `ipfs.files.cat`?

Reply:
>  offset and count

👍 

> ipfs.files.read

This is a MFS specific operation.

Reply:
> > offset and count
>
> 👍

Done